openmp/runtime/src/z_Linux_util.cpp

```diff
@@ -878,6 +878,15 @@ void __kmp_create_worker(int gtid, kmp_info_t *th, size_t stack_size) {
     KMP_SYSFAIL("pthread_create", status);
   }
 
+#ifdef KMP_OS_LINUX
+#if ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 12)))
+  // Rename worker threads for improved debuggability
+  if (!KMP_UBER_GTID(gtid)) {
+    pthread_setname_np(handle, "openmp_worker");
+  }
+#endif
+#endif
```

Comment:
You can simplify this
```suggestion
#if defined(KMP_OS_LINUX) && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 12)))
  // Rename worker threads for improved debuggability
  if (!KMP_UBER_GTID(gtid)) {
    pthread_setname_np(handle, "openmp_worker");
  }
#endif
```